### DISCUSSION TITLE
convert multi-layer .psd to a single .png containing all layers and preserving transparency

### DISCUSSION THREAD

**Discussion by 65c102** [ImageMagick/ImageMagick#3727] [Help]
what are the correct options for this ? everything I have tried fails

I either get an image containing all layers but the background is white and not transparent like the original
or I get multiple png files created with transparency one for each layer

Both results are not what I require

Please can someone help as I have search for information and tried every combination I can think of

Thanks

**Comment by fmw42** [comment]
Does your PSD image contain background transparency or an alpha channel? If the latter, then that cannot be done. You can only keep background transparency. Please post your PSD file. What was the exact command you tried? What is your IM version and platform?
